protocol_decoder/src/decoding.rs

```diff
@@ -350,12 +354,26 @@ impl ProcessedBlockTrace {
                 let txn_idx_of_dummy_entry =
                     txn_ir.last().unwrap().txn_number_before.low_u64() as usize + 1;
 
-                // Dummy state will be the state after the final txn.
-                let mut withdrawal_dummy =
-                    create_dummy_gen_input(other_data, extra_data, final_trie_state);
+                // To avoid double hashing the addrs, but I don't know if the extra `Vec`
+                // allocation is worth it.
```

Comment:
This comment should probably be removed.

Reply:
As there are a few places where the allocations may be removed, I think it could be preferable to keep this "for now", and remove it during the. Allocations PR, whether we find a concrete answer to this section or not. Or at least turn it as a todo